Observer/Admin/DashboardInstantiation.php

```diff
@@ -119,13 +119,15 @@ public function __construct(
     public function execute(Observer $observer) : void
     {
         try {
-            $store = $this->storeManager->getStore();
-
-            // TODO use $storeId
-            if ($this->config->isMerchantActive() || $this->config->getAccessToken()) {
+            $event = $observer->getEvent()->getData();
+            $store = $this->storeManager->getStore($event['storeId']);
+            if ($store === null) {
```

Comment:
For code cleanliness given this file has a fair amount of conditionals for the number of lines, would you be open to rewriting the store check as:
```
$store = $this->storeManager->getStore($event['storeId']) ?? $this->storeManager->getStore();
```